.vscodeignore

```diff
@@ -1,17 +1,44 @@
-.vscode/**
-.vscode-test/**
-out/test/**
-out/**/*.map
-out/**/*.ts
-src/**
-gif/**
-hooks/**
-test/**
-docs/**
+.vscode
+.vscode-test
+!node_modules/keytar
+!node_modules/getmac
+!node_modules/editions
+!node_modules/errlop
+!node_modules/extract-opts
+!node_modules/typechecker
+!node_modules/eachr
+!node_modules/getos
+!node_modules/node-usb-native/native
+!node_modules/usb-detection/build/Release/detection.node
+!node_modules/usb-detection/*.*
+!node_modules/eventemitter2
+!node_modules/bindings
+!node_modules/file-uri-to-path
+!node_modules/@serialport/bindings
+!node_modules/@serialport/binding-abstract
+!node_modules/@serialport/parser-readline
+!node_modules/@serialport/parser-delimiter
+!node_modules/debug
+!node_modules/ms
+node_modules
+out
+src
```

Comment:
Are we still have `src/` folder?

Reply:
No, this line actually exclude src folder. The old way is to retain the whole out/src folder, now we don't keep them all instead of one single .js under dist/ folder.